<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_select_another_test</class>
 <widget class="QDialog" name="Dialog_select_another_test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog_select_another_test{
border-image: url(:/new/prefix1/Diagnosis_Image/bg_mainview.png);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择要执行的测试：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_select_tool_1" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_tool_serch">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>测试搜索：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_tool_serch">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tool_serch">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CbTestPreference">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>只看信息熵</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>只看测试代价</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>信息熵优先</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>测试代价优先</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="Test_tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>测试点名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>信息熵</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>测试代价</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Button_OK">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_OK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_select_another_test</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_select_another_test</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
